values.yaml

```diff
@@ -14,6 +14,15 @@ podAnnotations: []
 nameOverride: ""
 fullnameOverride: ""
 
+podSecurityContext:
+  fsGroup: 13001
+  runAsUser: 13001
+
+securityContext:
+  runAsUser: 13001
+  runAsNonRoot: true
+  allowPrivilegeEscalation: false
```

Comment:
So by default we run a non root. Ok.

Reply:
It's already the case in fact. It's just that before the securityContext was hard coded in the template. I extracted it to the `values.yaml` so we can add more option.